pyconcz/templates/default/proposals/financial_aid_about.html

```diff
@@ -20,9 +20,9 @@ <h1>Financial Aid</h1>
         <h2>Timeline</h2>
 
         <p class="lead">
-            Financial aid applications open on 15 April and <b>close on 26 April</b>.
-            <br>After that we will carefully examine each application and
-            <br>inform all applicants privately before the end of April about the result.
+            Financial aid applications open on {{ proposal_config.date_start | date:'d F' }} and
+            <b>close on {{ proposal_config.date_end | date:'d F' }}</b>.
+            After that we will carefully examine each application and inform all applicants privately in the middle of May about the result.
```

Comment:
Just a nitpick you can replace the name of the month with {{ proposal_config.date_end | date:'F' }} so it will be taken from conf for the finaid.

Reply:
Fixed